src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
```

Comment:
Since `Random` is a dependency for the `Lottery` class, consider passing it through the constructor to make the class more testable and to avoid tight coupling with the `Random` implementation.